js/views/modals/editListing/EditListing.js

```diff
@@ -875,18 +837,42 @@ export default class extends BaseModal {
       this._$buttonSave = null;
       this._$inputPhotoUpload = null;
       this._$photoUploadingLabel = null;
-      this._$photoUploadItems = null;
       this._$editListingReturnPolicy = null;
       this._$editListingTermsAndConditions = null;
       this._$sectionShipping = null;
       this._$maxCatsWarning = null;
       this._$maxTagsWarning = null;
       this._$addShipOptSectionHeading = null;
-      this._$closeClickTargets = null;
+      this.$photoUploadItems = this.$('.js-photoUploadItems');
       this.$modalContent = this.$('.modalContent');
       this.$tabControls = this.$('.tabControls');
       this.$titleInput = this.$('#editListingTitle');
 
+      installRichEditor(this.$('#editListingDescription'), {
+        topLevelClass: 'clrBr',
+      });
+
+      if (this.sortablePhotos) this.sortablePhotos.destroy();
+      this.sortablePhotos = Sortable.create(this.$photoUploadItems[0], {
+        filter: '.js-addPhotoWrap',
+        onUpdate: (e) => {
+          const imageModels = this.innerListing
+            .get('item')
+            .get('images')
+            .models;
+
+          imageModels[e.oldIndex - 1] =
+            imageModels.splice(e.newIndex - 1, 1, imageModels[e.oldIndex - 1])[0];
+        },
+      });
```

Comment:
The sortable code lets you move the add photo button, it seems like that should be excluded. 

![image](https://cloud.githubusercontent.com/assets/1584275/21780384/649231a4-d679-11e6-837f-3eb04713f21f.png)